assets/src/scripts/components/map/index.js

```diff
@@ -20,11 +20,29 @@ const getLayerDefs = function(layerNo, siteno, stage) {
  * Creates a site map
  */
 const siteMap = function(node, {siteno, latitude, longitude, zoom}) {
+
+    var gray = layerGroup();
+    basemapLayer('Gray').addTo(gray);
+
+    var cityIcon = icon({
+        iconUrl: config.STATIC_URL + '/img/marker-icon-black.png'
+    });
+
+    var cities = featureLayer({
+        url: 'https://services.arcgis.com/P3ePLMYs2RVChkJx/arcgis/rest/services/USA_Major_Cities/FeatureServer/0',
```

Comment:
We typically put endpoints in config. So this should go in config.py and then added to the javascript global CONFIG which you can find in wdfn-server/waterdata/templates/base_plain.html.

Reply:
fixed